mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltaksgjennomforingRepository.kt

```diff
@@ -453,6 +457,7 @@ class TiltaksgjennomforingRepository(private val db: Database) {
             navn = string("navn"),
             tiltaksnummer = stringOrNull("tiltaksnummer"),
             virksomhetsnummer = string("virksomhetsnummer"),
+            virksomhetsnavn = stringOrNull("virksomhetsnavn"),
```

Comment:
Dadde det egnet seg å embedde en lettvariant av `VirksomhetDto` på `TiltaksgjennomforingAdminDto` tilsvarende det vi gjør for `tiltakstype` og `navEnheter` her?